### PR TITLE
GPII-3739: Open USB Drive

### PR DESCRIPTION
**Pull request by Karadaliev**
* created translations for the button and his tooltip (en and bg)
* added button's definition in settings.json (all the system buttons are now with tab-index 200+)
* added "usb-open" in the handlerGrades
* added openUSBButtonPresenter() handler for the USB, temporary just opening explorer.exe

### CODE REVIEW

**Comment by Karadaliev** [comment]
@amb26 Keep in mind that the current execution (just opening My Computer) its temporary, still waiting on @stegru implementation. Still you can take a look if everything else its up to snuff ;-).

**Comment by Karadaliev** [comment]
@amb26 Can you still take a look at the task at its current state, it will be great to know if everything else (except opening the right USB drive) its alright. Thanks in advance!

**Comment by stegru** [comment]
ok to test

**Comment by stegru** [comment]
@Karadaliev, I'm working on this now. Could you update and resolve these conflicts?

**Comment by Karadaliev** [comment]
> @Karadaliev, I'm working on this now. Could you update and resolve these conflicts?
Yup, I will merge it with the current master and resolve all conflicts. Probably tomorrow about noon. Thanks!

**Comment by stegru** [comment]
Would you like to merge my branch? https://github.com/stegru/gpii-app/tree/GPII-3739

https://github.com/krisYanachkov/gpii-app/compare/GPII-3739...stegru:GPII-3739

For some reason, the UI isn't working for me so I haven't ran the code yet.

File: tests/QssTestDefs.js [hunk 1]
```diff
@@ -1362,6 +1400,12 @@ var quickFoldersTestSequence = [
         ],
         resolve: "jqUnit.assertTrue",
         resolveArgs: ["The quick folder button is displayed: ", "{arguments}.0"]
+    }, {   // XXX we need some minor timeout for the QSS to get
+        // in normal state. In case this is not present,
+        // the next item doesn't take effect
+        task: "gpii.test.linger",
```
Comment (amb26) [inline tests/QssTestDefs.js]: Please figure out what event you need to wait for and add a listener for it instead of this long timeout

**Comment by amb26** [comment]
Hi there @Karadaliev - could you resolve conflicts with this one - also, doesn't it need to depend on some update to the gpii-windows project?

**Comment by Karadaliev** [comment]
> Hi there @Karadaliev - could you resolve conflicts with this one - also, doesn't it need to depend on some update to the gpii-windows project?

Yes, I merged the changes just to be able to test them out. And this depends on gpii-windows update for sure (I'm using Steve's PR for now), but on top of that its not working at all. Because in the renderer process I don't have access to the gpii.windows at all 🌵 . So we will need to talk about it, maybe I will create a custom even that I will envoke through the notifier listener or something... I will start looking into it tomorrow, and will keep you in the loop.

File: tests/QssTestDefs.js (hunk 1 above)
Comment (amb26) [reply tests/QssTestDefs.js]: Bump comment

File: src/renderer/qssWidget/js/qssWidget.js [hunk 2]
```diff
@@ -71,6 +71,7 @@
         events: {
             onWidgetClosed: null,
             onQssWidgetHideQssRequested: null,
+            onQssOpenUsbRequested: null,
```
Comment (amb26) [inline src/renderer/qssWidget/js/qssWidget.js]: My suspicion is that these updates to qssWidget.js are unnecessary - could you experiment with removing them and report what happens?

Comment (Karadaliev) [reply src/renderer/qssWidget/js/qssWidget.js]: Yup, you are totally right about this, it works without the definition here.

File: tests/QssTestDefs.js (hunk 1 above)
Comment (Karadaliev) [reply tests/QssTestDefs.js]: @krisYanachkov can you take a look on this, because its the last part of this PR.

**Comment by Karadaliev** [comment]
@amb26 are we waiting on something in order this PR to be merged? If so let us know, because for now looks like its ready to be merged.

**Pull request merged**